blog/_posts/2018-11-09-dvteam-farewell

```diff
@@ -0,0 +1,134 @@
+---
+title: DV Team Farewell
+layout: post
+created_at: Fri Sep 7 2018
+permalink: blog/2018-11-09-dvteam-farewell
+current: blog
+author: Violeta & Diana
+twitter: artnerdnet
+categories: student-posts
```

Comment:
To be consistent with other files, add the declaration of student-posts in the next line. You can check other files for the example.